click_helpers.py

```diff
@@ -147,8 +147,7 @@ def click_to_middle(  # pylint: disable=too-many-arguments
         bool: True if successfully clicked, False otherwise.
     """
     path = resource_path(path)
-    pos = find_image(path, precision)
-    if pos:
+    if pos := find_image(path, precision):
```

Comment:
I personally do not find this to improve readability and actually find it reduces readability, since you're performing an assignment at the same time as a conditional check. (this applies elsewhere in this PR, too)